03/index.html

```diff
@@ -5,12 +5,32 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>devmentor.pl - HTML & CSS: RWD - #03</title>
+    <link rel="stylesheet" href="./styles/global.css">
+    <!--<link rel="stylesheet" href="./styles/mobile.css">-->
+    <!--<link rel="stylesheet" href="./styles/tablet.css">-->
```

Comment:
Nie komentuj sobie styli bo utrudniasz sobie pracę.
Potem w docelowej wersji będziesz musiał od komentować i okaże się, że style nachodzą na siebie i wszystko przestaje działać.
Zdecydowanie lepszym rozwiązaniem jest określenie atrybutu media np. `media="(min-width: 699px) and (max-width: 1299px)"`

Reply:
Ten atrybut mam wpisać w body ?

Reply:
Ten atrybut wpisujesz do `link`

Reply:
czyli jako @media tak ?

Reply:
To powinno być w jednym css tak ? Np w tym global ?

Reply:
Możesz zrobić coś takiego:
```
<link rel="stylesheet" href="./styles/mobile.css">
<link rel="stylesheet" media="(min-width: 700px)" href="./styles/tablet.css">
```
Przy tym zapisie style w pliku mobile.css będą zawsze dostępne, a z pliku `tablet.css` tylko dla rozdzielczości równej lub powyżej 700px